requirements.txt

```diff
@@ -10,5 +10,6 @@ pytest-cov
 python-dotenv
 pandas
 pvsite-datamodel==0.1.25
+sentry-sdk
```

Comment:
This file should have been removed when I switched to `poetry` (my bad!), I think you could remove it altogether.